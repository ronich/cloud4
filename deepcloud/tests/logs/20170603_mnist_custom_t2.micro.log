Downloading data from https://s3.amazonaws.com/img-datasets/mnist.npz
    8192/11490434 [..............................] - ETA: 0s 1138688/11490434 [=>............................] - ETA: 0s 2416640/11490434 [=====>........................] - ETA: 0s 3743744/11490434 [========>.....................] - ETA: 0s 5038080/11490434 [============>.................] - ETA: 0s 6283264/11490434 [===============>..............] - ETA: 0s 7397376/11490434 [==================>...........] - ETA: 0s 8806400/11490434 [=====================>........] - ETA: 0s10788864/11490434 [===========================>..] - ETA: 0sEpoch 1/36
epoch ended:386.70591378211975
[10.312445402145386, 0.3772139549255371, 0.37880396842956543, 0.37635302543640137, 0.37754154205322266, 0.3768303394317627, 0.3781247138977051, 0.37713146209716797, 0.37866997718811035, 0.3767666816711426, 0.376816987991333, 0.3837251663208008, 0.3922419548034668, 0.390704870223999, 0.3845787048339844, 0.38480472564697266, 0.38553619384765625, 0.38822507858276367, 0.4022679328918457, 0.38576412200927734, 0.38532280921936035, 0.3871588706970215, 0.521214485168457, 0.38234639167785645, 0.3854224681854248, 0.383591890335083, 0.38436317443847656, 0.40189552307128906, 0.40987110137939453, 0.53200364112854, 0.5894172191619873, 0.3829183578491211, 0.38365793228149414, 0.38227176666259766, 0.3834404945373535, 0.41673803329467773, 0.38105297088623047, 0.3809969425201416, 0.3798091411590576, 0.3809947967529297, 0.3824446201324463, 0.3827509880065918, 0.384213924407959, 0.3819553852081299, 0.38208985328674316, 0.38196730613708496, 0.38228797912597656, 0.38207530975341797, 0.3830738067626953, 0.3829953670501709, 0.38248229026794434, 0.3838467597961426, 0.3831007480621338, 0.38274049758911133, 0.38303303718566895, 0.3835265636444092, 0.3835427761077881, 0.3833026885986328, 0.3822345733642578, 0.38121581077575684, 0.382185697555542, 0.3819906711578369, 0.3815035820007324, 0.38349127769470215, 0.38315606117248535, 0.38291120529174805, 0.3843700885772705, 0.3832225799560547, 0.3828396797180176, 0.3838527202606201, 0.3829927444458008, 0.3829917907714844, 0.3832571506500244, 0.382901668548584, 0.3842794895172119, 0.3824458122253418, 0.3826432228088379, 0.383620023727417, 0.3829207420349121, 0.38399648666381836, 0.3831770420074463, 0.38260602951049805, 0.3815152645111084, 0.381483793258667, 0.3799021244049072, 0.380237340927124, 0.3810267448425293, 0.3803575038909912, 0.3800516128540039, 0.38213682174682617, 0.38077878952026367, 0.39707303047180176, 0.3818700313568115, 0.38138747215270996, 0.3836214542388916, 0.38164234161376953, 0.38320016860961914, 0.38259029388427734, 0.3810086250305176, 0.38257455825805664, 0.3834807872772217, 0.3829951286315918, 0.3836045265197754, 0.3836796283721924, 0.38387155532836914, 0.3843810558319092, 0.3854804039001465, 0.3849759101867676, 0.38298821449279785, 0.3823978900909424, 0.38160109519958496, 0.38204360008239746, 0.3828773498535156, 0.3835630416870117, 0.38243722915649414, 0.3833942413330078, 0.38373661041259766, 0.38317275047302246, 0.38245558738708496, 0.3847537040710449, 0.383664608001709, 0.3844418525695801, 0.3850228786468506, 0.3864095211029053, 0.3861353397369385, 0.38638901710510254, 0.3859546184539795, 0.3850088119506836, 0.38288021087646484, 0.382828950881958, 0.3770606517791748, 0.3763577938079834, 0.3781445026397705, 0.37722206115722656, 0.37552714347839355, 0.3781440258026123, 0.37915992736816406, 0.37819838523864746, 0.3778700828552246, 0.37824487686157227, 0.37163305282592773, 0.3784303665161133, 0.381666898727417, 0.3781766891479492, 0.38307952880859375, 0.38451075553894043, 0.3820993900299072, 0.38568735122680664, 0.38196778297424316, 0.38308072090148926, 0.3817150592803955, 0.3787877559661865, 0.38025593757629395, 0.3851490020751953, 0.38335442543029785, 0.38091421127319336, 0.3826436996459961, 0.3833303451538086, 0.38232922554016113, 0.3801589012145996, 0.38242483139038086, 0.37807440757751465, 0.3798856735229492, 0.3775138854980469, 0.38268399238586426, 0.3811488151550293, 0.37801408767700195, 0.3797340393066406, 0.38268589973449707, 0.3802378177642822, 0.37473583221435547, 0.38007664680480957, 0.3756415843963623, 0.38051843643188477, 0.3767225742340088, 0.37576889991760254, 0.3784177303314209, 0.3744163513183594, 0.37279820442199707, 0.3749668598175049, 0.383469820022583, 0.37938833236694336, 0.3843863010406494, 0.3855268955230713, 0.38435959815979004, 0.38492250442504883, 0.3843116760253906, 0.3837308883666992, 0.3850529193878174, 0.3838808536529541, 0.3832688331604004, 0.3798401355743408, 0.38002610206604004, 0.3741798400878906, 0.37369346618652344, 0.373382568359375, 0.3826932907104492, 0.3815727233886719, 0.3765714168548584, 0.382845401763916, 0.3880002498626709, 0.39825868606567383, 0.37899208068847656, 0.36952805519104004, 0.3783383369445801, 0.7051846981048584, 0.49010252952575684, 0.38300108909606934, 0.4203794002532959, 0.38500046730041504, 0.3834230899810791, 0.3842449188232422, 0.3789234161376953, 0.38449954986572266, 0.38439154624938965, 0.3828585147857666, 0.385941743850708, 0.38452816009521484, 0.3826925754547119, 0.38617539405822754, 0.3866426944732666, 0.3794286251068115, 0.3826138973236084, 0.38187265396118164, 0.38340234756469727, 0.3832414150238037, 0.38244128227233887, 0.38106489181518555, 0.3861992359161377, 0.382767915725708, 0.3844881057739258, 0.38684725761413574, 0.3856537342071533, 0.3854808807373047, 0.3832244873046875, 0.3826792240142822, 0.3821086883544922, 0.3834357261657715, 0.3838372230529785, 0.38170480728149414, 0.3851175308227539, 0.3793308734893799, 0.3817880153656006, 0.3818211555480957, 0.38272762298583984, 0.38294100761413574, 0.38160061836242676, 0.37889599800109863, 0.3777010440826416, 0.37990474700927734, 0.38150644302368164, 0.3768918514251709, 0.37644410133361816, 0.378495454788208, 0.37690281867980957, 0.37447357177734375, 0.37641119956970215, 0.3817152976989746, 0.3803853988647461, 0.3792128562927246, 0.37812352180480957, 0.38286328315734863, 0.37411046028137207, 0.37992000579833984, 0.3773221969604492, 0.37955808639526367, 0.37583231925964355, 0.3794596195220947, 0.3769552707672119, 0.3809223175048828, 0.3745543956756592, 0.3738725185394287, 0.3808748722076416, 0.3773918151855469, 0.37678098678588867, 0.3789358139038086, 0.3799467086791992, 0.37938475608825684, 0.38018155097961426, 0.3738067150115967, 0.3798553943634033, 0.3840782642364502, 0.38311171531677246, 0.3853428363800049, 0.38378143310546875, 0.38190126419067383, 0.3788154125213623, 0.3734433650970459, 0.3785228729248047, 0.37543225288391113, 0.3780233860015869, 0.3759019374847412, 0.37880563735961914, 0.3811776638031006, 0.38323307037353516, 0.3739645481109619, 0.3730194568634033, 0.37877440452575684, 0.3773784637451172, 0.3776118755340576, 0.3818192481994629, 0.38191747665405273, 0.3832671642303467, 0.3839592933654785, 0.38663148880004883, 0.3843700885772705, 0.380950927734375, 0.3834829330444336, 0.38479042053222656, 0.3830559253692627, 0.3844740390777588, 0.3852958679199219, 0.38456177711486816, 0.3858363628387451, 0.38362860679626465, 0.38548851013183594, 0.3856518268585205, 0.3844318389892578, 0.38519287109375, 0.3848137855529785, 0.38181495666503906, 0.3792839050292969, 0.3817627429962158, 0.3793163299560547, 0.38098692893981934, 0.37959718704223633, 0.38184022903442383, 0.3825376033782959, 0.3818697929382324, 0.38485193252563477, 0.38154125213623047, 0.384735107421875, 0.38617610931396484, 0.3827190399169922, 0.3829350471496582, 0.3848848342895508, 0.3852052688598633, 0.3806285858154297, 0.38559460639953613, 0.4008934497833252, 0.38957977294921875, 0.38642144203186035, 0.38623952865600586, 0.3846306800842285, 0.3853139877319336, 0.38570618629455566, 0.3857114315032959, 0.38309454917907715, 0.38257455825805664, 0.38327980041503906, 0.38483214378356934, 0.3862273693084717, 0.3857870101928711, 0.3857686519622803, 0.38501882553100586, 0.3848252296447754, 0.383272647857666, 0.38370251655578613, 0.38198065757751465, 0.38292670249938965, 0.3852379322052002, 0.3840641975402832, 0.3847029209136963, 0.388416051864624, 0.3849964141845703, 0.38698339462280273, 0.38927412033081055, 0.38762831687927246, 0.38619208335876465, 0.394909143447876, 0.39131975173950195, 0.3874528408050537, 0.3862464427947998, 0.38756251335144043, 0.40244340896606445, 0.38544702529907227, 0.38439106941223145, 0.40167665481567383, 0.7421064376831055, 0.4562499523162842, 0.38582396507263184, 0.41884899139404297, 0.3833320140838623, 0.3833303451538086, 0.38271427154541016, 0.382749080657959, 0.38419461250305176, 0.383281946182251, 0.3813669681549072, 0.3827533721923828, 0.38175058364868164, 0.3828401565551758, 0.37932610511779785, 0.383573055267334, 0.3823070526123047, 0.38319921493530273, 0.3845088481903076, 0.3800334930419922, 0.38379454612731934, 0.3860819339752197, 0.39345884323120117, 0.3842320442199707, 0.3844740390777588, 0.38640928268432617, 0.38533759117126465, 0.38455677032470703, 0.3844718933105469, 0.3850390911102295, 0.3866007328033447, 0.38587236404418945, 0.38481664657592773, 0.38618898391723633, 0.3858201503753662, 0.3871040344238281, 0.38402509689331055, 0.3818399906158447, 0.385669469833374, 0.3858184814453125, 0.3835926055908203, 0.38646578788757324, 0.38411927223205566, 0.3824803829193115, 0.3814394474029541, 0.3823208808898926, 0.37815284729003906, 0.3802530765533447, 0.38290834426879883, 0.38292551040649414, 0.3827025890350342, 0.38300514221191406, 0.37799978256225586, 0.38291049003601074, 0.3836402893066406, 0.3823843002319336, 0.3834540843963623, 0.3842189311981201, 0.3846926689147949, 0.3851954936981201, 0.38491153717041016, 0.3833334445953369, 0.38358378410339355, 0.37824416160583496, 0.3751356601715088, 0.375154972076416, 0.37590742111206055, 0.3740663528442383, 0.37378859519958496, 0.3733482360839844, 0.37459492683410645, 0.3767075538635254, 0.37338995933532715, 0.37482118606567383, 0.3723106384277344, 0.37949085235595703, 0.3722653388977051, 0.37421655654907227, 0.3791313171386719, 0.3722255229949951, 0.37409377098083496, 0.37451910972595215, 0.37577319145202637, 0.37557077407836914, 0.3760807514190674, 0.38155531883239746, 0.3822293281555176, 0.37803053855895996, 0.3730316162109375, 0.3768613338470459, 0.382796049118042, 0.3829765319824219, 0.3819441795349121, 0.37839698791503906, 0.3816201686859131, 0.3779435157775879, 0.3777897357940674, 0.3777146339416504, 0.37895703315734863, 0.3782532215118408, 0.3767824172973633, 0.3810610771179199, 0.37790989875793457, 0.37615180015563965, 0.3772897720336914, 0.3755478858947754, 0.3766026496887207, 0.38031530380249023, 0.3753087520599365, 0.3859877586364746, 0.3819406032562256, 0.37776970863342285, 0.3758978843688965, 0.3802988529205322, 0.3796672821044922, 0.37959885597229004, 0.38654041290283203, 0.3848402500152588, 0.3853495121002197, 0.38695192337036133, 0.3868539333343506, 0.38569188117980957, 0.38702917098999023, 0.38550519943237305, 0.38441967964172363, 0.3852658271789551, 0.38289380073547363, 0.38144421577453613, 0.3836817741394043, 0.3822007179260254, 0.3786308765411377, 0.3784635066986084, 0.37568044662475586, 0.37917542457580566, 0.38193535804748535, 0.38184666633605957, 0.3851327896118164, 0.38395261764526367, 0.3830111026763916, 0.3836038112640381, 0.3842484951019287, 0.38455724716186523, 0.3852972984313965, 0.3839530944824219, 0.38341689109802246, 0.38039159774780273, 0.3804969787597656, 0.3853285312652588, 0.38141918182373047, 0.38424205780029297, 0.3803677558898926, 0.3787062168121338, 0.3754608631134033, 0.370333194732666, 0.37774205207824707, 0.37479329109191895, 0.37192511558532715, 0.3808777332305908, 0.37491822242736816, 0.37943363189697266, 0.3771946430206299, 0.37512993812561035, 0.37309813499450684, 0.374814510345459, 0.37676310539245605, 0.38616347312927246, 0.38129496574401855, 0.3814244270324707, 0.38253331184387207, 0.38147974014282227, 0.39786839485168457, 0.38355493545532227, 0.38356685638427734, 0.38466596603393555, 0.7122941017150879, 0.47615885734558105, 0.373676061630249, 0.4082028865814209, 0.37061119079589844, 0.3712785243988037, 0.37584733963012695, 0.37674665451049805, 0.37674450874328613, 0.3745734691619873, 0.3806731700897217, 0.38149452209472656, 0.38214635848999023, 0.3819911479949951, 0.38044071197509766, 0.3801119327545166, 0.377699613571167, 0.3802769184112549, 0.38320064544677734, 0.38132309913635254, 0.38489818572998047, 0.38411998748779297, 0.38309407234191895, 0.3819539546966553, 0.3846781253814697, 0.3831818103790283, 0.38463854789733887, 0.3829312324523926, 0.383225679397583, 0.3848891258239746, 0.379335880279541, 0.38300156593322754, 0.38457179069519043, 0.3831610679626465, 0.38492655754089355, 0.3841888904571533, 0.3850977420806885, 0.38132739067077637, 0.3834340572357178, 0.3858036994934082, 0.3855454921722412, 0.3857858180999756, 0.3853268623352051, 0.3852400779724121, 0.3850986957550049, 0.38544464111328125, 0.38605189323425293, 0.3851194381713867, 0.38596677780151367, 0.38470911979675293, 0.3852527141571045, 0.38437557220458984, 0.38390135765075684, 0.3836503028869629, 0.38370323181152344, 0.38474297523498535, 0.38017988204956055, 0.3845365047454834, 0.3830718994140625, 0.37419795989990234, 0.37625765800476074, 0.37502193450927734, 0.3747391700744629, 0.3765392303466797, 0.374922513961792, 0.37743663787841797, 0.37839555740356445, 0.3759315013885498, 0.3753654956817627, 0.3759799003601074, 0.3779573440551758, 0.3771655559539795, 0.37488412857055664, 0.37811970710754395, 0.37194299697875977, 0.37639737129211426, 0.37537431716918945, 0.37249302864074707, 0.3758890628814697, 0.3754305839538574, 0.37546777725219727, 0.37517380714416504, 0.38098812103271484, 0.3801865577697754, 0.37419891357421875, 0.37445974349975586, 0.3772709369659424, 0.3772432804107666, 0.3762037754058838, 0.3770291805267334, 0.37461400032043457, 0.3746318817138672, 0.3780045509338379, 0.3755331039428711, 0.3760077953338623, 0.380082368850708, 0.38110995292663574, 0.3775765895843506, 0.3822197914123535, 0.385408878326416, 0.38651371002197266, 0.3844430446624756, 0.3842802047729492, 0.3855423927307129, 0.38297462463378906, 0.38226866722106934, 0.3826632499694824, 0.3855128288269043, 0.38479185104370117, 0.385894775390625, 0.38532114028930664, 0.3858976364135742, 0.3860902786254883, 0.384843111038208, 0.386646032333374, 0.3841254711151123, 0.3858664035797119, 0.3851280212402344, 0.38550233840942383, 0.38552117347717285, 0.3850886821746826, 0.38518834114074707, 0.3875298500061035, 0.3867480754852295, 0.3864915370941162, 0.38765764236450195, 0.38771986961364746, 0.3865213394165039, 0.3857593536376953, 0.3863868713378906, 0.3868415355682373, 0.38732218742370605, 0.3858616352081299, 0.3848872184753418, 0.3837697505950928, 0.3856387138366699, 0.3861382007598877, 0.3866286277770996, 0.38519763946533203, 0.3867955207824707, 0.38546299934387207, 0.38468289375305176, 0.3858506679534912, 0.3854501247406006, 0.3859519958496094, 0.38529133796691895, 0.38576722145080566, 0.3852083683013916, 0.3855865001678467, 0.38491249084472656, 0.38501501083374023, 0.38346076011657715, 0.3838365077972412, 0.3837010860443115, 0.3835721015930176, 0.3786041736602783, 0.38111329078674316, 0.38219308853149414, 0.3841874599456787, 0.3813154697418213, 0.3823204040527344, 0.38473010063171387, 0.38554930686950684, 0.39166784286499023, 0.3886713981628418, 0.38688182830810547, 0.38759350776672363, 0.3875465393066406, 0.3888366222381592, 0.40186548233032227, 0.38230228424072266, 0.38446664810180664, 0.6303043365478516, 0.5723011493682861, 0.3854942321777344, 0.4125092029571533, 0.3806276321411133, 0.3765997886657715, 0.3803730010986328, 0.38103175163269043, 0.38212060928344727, 0.3814358711242676, 0.37828493118286133, 0.38236355781555176, 0.38567018508911133, 0.38274717330932617, 0.3789980411529541, 0.3800818920135498, 0.38341498374938965, 0.37737369537353516, 0.379744291305542, 0.37586426734924316, 0.3811831474304199, 0.38391685485839844, 0.3839993476867676, 0.3858530521392822, 0.38531494140625, 0.3850259780883789, 0.38431262969970703, 0.38546228408813477, 0.385540246963501, 0.3838932514190674, 0.38205862045288086, 0.3804798126220703, 0.3782329559326172, 0.3813328742980957, 0.38267016410827637, 0.3806722164154053, 0.37818217277526855, 0.3834652900695801, 0.38063883781433105, 0.38290858268737793, 0.37691235542297363, 0.38313889503479004, 0.37903809547424316, 0.3794748783111572, 0.38004231452941895, 0.37677741050720215, 0.3761875629425049, 0.37777018547058105, 0.37575411796569824, 0.37995481491088867, 0.3770942687988281, 0.3789987564086914, 0.3809983730316162, 0.38167238235473633, 0.3781261444091797, 0.3780388832092285, 0.3786327838897705, 0.38170957565307617, 0.3846306800842285, 0.3852388858795166, 0.38588476181030273, 0.3860962390899658, 0.3864471912384033, 0.3855423927307129, 0.3856322765350342, 0.3847970962524414, 0.3834562301635742, 0.38475704193115234, 0.3862590789794922, 0.3868539333343506, 0.38512301445007324, 0.38391566276550293, 0.3817136287689209, 0.38247084617614746, 0.3840305805206299, 0.37819647789001465, 0.37549853324890137, 0.37752842903137207, 0.3759610652923584, 0.383197546005249, 0.38462138175964355, 0.3851053714752197, 0.38558101654052734, 0.38571763038635254, 0.38529157638549805, 0.3841683864593506, 0.382918119430542, 0.3844747543334961, 0.38373470306396484, 0.38460850715637207, 0.38402271270751953, 0.38477349281311035, 0.3851187229156494, 0.38297438621520996, 0.38532090187072754, 0.3839149475097656, 0.38579487800598145, 0.3855099678039551, 0.3852555751800537, 0.38776350021362305, 0.3856034278869629, 0.3851447105407715, 0.3872537612915039, 0.38655638694763184, 0.3867650032043457, 0.38483619689941406, 0.3875563144683838, 0.3854045867919922, 0.38495326042175293, 0.38570666313171387, 0.3843977451324463, 0.3845553398132324, 0.3853118419647217, 0.3845400810241699, 0.3855140209197998, 0.38520026206970215, 0.3852832317352295, 0.3849015235900879, 0.38597607612609863, 0.3889145851135254, 0.3873941898345947, 0.385678768157959, 0.38296961784362793, 0.38301944732666016, 0.3832716941833496, 0.3835325241088867, 0.3837711811065674, 0.38231849670410156, 0.38327980041503906, 0.38166356086730957, 0.3822147846221924, 0.3829195499420166, 0.38399219512939453, 0.3846089839935303, 0.3825552463531494, 0.3831472396850586, 0.38535022735595703, 0.3828542232513428, 0.3835749626159668, 0.38222408294677734, 0.3834044933319092, 0.38425469398498535, 0.3836188316345215, 0.381572961807251, 0.37983059883117676, 0.3823978900909424, 0.3842175006866455, 0.38472461700439453, 0.38286757469177246, 0.38474178314208984, 0.3855617046356201, 0.38492679595947266, 0.3828444480895996, 0.38442039489746094, 0.384258508682251, 0.38369059562683105, 0.38376760482788086, 0.38183045387268066, 0.3817873001098633, 0.38144683837890625, 0.384105920791626, 0.3843827247619629, 0.38374924659729004, 0.3824634552001953, 0.38254833221435547, 0.38892483711242676, 0.3883190155029297, 0.3817439079284668, 0.3842465877532959, 0.38685107231140137, 0.38665246963500977, 0.3992276191711426, 0.3855586051940918, 0.3840668201446533, 0.3923227787017822, 0.7195854187011719, 0.4664123058319092, 0.3847784996032715, 0.4175839424133301, 0.3774552345275879, 0.3752007484436035, 0.38497185707092285, 0.3796505928039551, 0.38265538215637207, 0.38486361503601074, 0.38259243965148926, 0.3825688362121582, 0.3817119598388672, 0.3751504421234131, 0.3701186180114746, 0.3737924098968506, 0.3813655376434326, 0.38094115257263184, 0.37653589248657227, 0.3748607635498047, 0.3739917278289795, 0.37841033935546875, 0.38059544563293457, 0.38015317916870117, 0.380462646484375, 0.38338685035705566, 0.3793509006500244, 0.37836480140686035, 0.37537479400634766, 0.3841979503631592, 0.38021159172058105, 0.3802609443664551, 0.3766663074493408, 0.3793778419494629, 0.3782486915588379, 0.3792130947113037, 0.37758922576904297, 0.38338303565979004]
386s - loss: 1.6921 - acc: 0.4066 - val_loss: 0.5399 - val_acc: 0.8825
Epoch 2/36
