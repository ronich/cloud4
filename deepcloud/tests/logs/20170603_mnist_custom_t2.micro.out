epoch,acc,loss,val_acc,val_loss
0,0.406566835646,1.69210000284,0.8825,0.539928189707
